pkgs/by-name/li/little_boxes/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, installShellFiles
+, callPackage
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "little_boxes";
+  version = "1.10.0";
+
+  src = fetchFromGitHub {
+    owner = "giodamelio";
+    repo = "little_boxes";
+    rev = version;
+    hash = "sha256-Quh09K5meiA39ih/orJWF2WfkuZdymxub1dZvns/q3E=";
+  };
+
+  cargoVendorDir = "vendor";
+
+  # Install manpages/completions
+  nativeBuildInputs = [ installShellFiles ];
+  postInstall = ''
+    extrasPath=$(ls -d target/*/release/build/little_boxes-*/out/)
+
+    installManPage $extrasPath/little_boxes.1
+    installShellCompletion --bash $extrasPath/little_boxes.bash
+    installShellCompletion --fish $extrasPath/little_boxes.fish
+    installShellCompletion --zsh $extrasPath/_little_boxes
+  '';
+
+  passthru.tests.version = callPackage ./test-version.nix {};
+
+  meta = with lib; {
+    description = "Put your things in boxes";
```

The short `description` is a little mysterious but it's fine.
Can you add a `longDescription`?

P.S.: to be honest, by reading the upstream repo, I think a better description would be 

```suggestion
    description = "Adds boxes around input text";
```

But this is at your discretion.